tools/caffe/caffe2ncnn.cpp

```diff
@@ -321,18 +321,23 @@ static bool read_proto_from_binary(const char* filepath, google::protobuf::Messa
 
 int main(int argc, char** argv)
 {
-    if (!(argc == 3 || argc == 5 || argc == 6 || argc == 7))
+    int ocr = 0;
+
+    if (!(argc == 4 || argc == 6 || argc == 7 || argc == 8))
     {
-        fprintf(stderr, "Usage: %s [caffeproto] [caffemodel] [ncnnproto] [ncnnbin] [quantizelevel] [int8scaletable]\n", argv[0]);
+        fprintf(stderr, "Usage: %s ocr|noocr [caffeproto] [caffemodel] [ncnnproto] [ncnnbin] [quantizelevel] [int8scaletable]\n", argv[0]);
```

Comment:
new optional parameter should always be placed as the last one for keeping the old usage works